qwik-graphql-tailwind/src/components/pull-request-issue-tab/pull-request-issue-tab.tsx

```diff
@@ -62,7 +67,7 @@ export const PullRequestIssueTab = component$(
         <div class="flex items-center space-x-8">
           <div>
             <FilterDropdown name="Label" description="Filter by label" buttonClassName="border-none text-sm">
-              {labelOption.map(({ label, value, color, description }) => (
+              {labelOption?.map(({ label, value, color, description }) => (
```

Comment:
I think this question mark is unnecessary. We always have the labelOption array (empty or with values)

Reply:
The reason i didn't comment on this was because when declaring the type labelOption can be optional. Not sure why.

Reply:
In PR we get the label options from the API, so we don't always have the option.